packages/react-components/src/Toggle.tsx

```diff
@@ -52,6 +61,21 @@ const StyledDiv = styled.div`
     vertical-align: middle;
   }
 
+  .ui--Switch {
```

Comment:
As I've already suggested maybe we can shift the button a bit higher. Not critical, until we have a new design , but will help readability of the URL.